packages/framework/assets/js/admin/components/GridInlineEdit.js

```diff
@@ -15,7 +15,7 @@ export default class GridInlineEdit {
         $grid
             .off('click', '.js-inline-edit-edit')
             .on('click', '.js-inline-edit-edit', function () {
-                const $row = $(this).closest('.js-grid-row');
+                const $row = $(this).closest('.test-grid-row');
```

Comment:
I thought the idea is to have test-* classes for acceptance testing and I see it's used also in the application.
For me personally, it's now confusing (like should I use `test-`, `js-`, or without in the future code. 

Isn't better to have `test-` classes for acceptance tests exclusively and keep `js-` for javascript functionality? What do you think?

Reply:
Well in my point of view it is better to have separate classes for tests and javascripts. It is because javascript behavior can be changed or test behavior can be changed and you do not want to impact other one.

Reply:
I agree completely :)
That's what I'm asking about. it seems to me, like in this JS file `js-grid-row` should be used as it's not related to tests, but to functionality, or am I mistaken?

Reply:
d0fff7e891441d2e7d60375363b118981ccb0a89